### PR TITLE
Add spanish guarani dictionary app to app store.

### PR DESCRIPTION
**Pull request by rmacqueen**
https://phabricator.endlessm.com/T12825

### CODE REVIEW

File: content/Default/apps/content.json [hunk 1]
```diff
@@ -2031,6 +2031,44 @@
...
+      "pt": [
+        "com.endlessm.spanish_guarani_dict-screenshot1.jpg",
+        "com.endlessm.spanish_guarani_dict-screenshot2.jpg",
+        "com.endlessm.spanish_guarani_dict-screenshot3.jpg"
+      ]
+    },
+    "splash-screen-type": "Custom",
+    "square_img": "com.endlessm.spanish_guarani_dict-thumb.jpg",
+    "subtitle": "Diccionario bilingüe de Castellano-Guaraní y Guaraní-Castellano",
+    "title": "Diccionario Bilingüe | Ñe’ẽ Ayvu",
```
Comment (ptomato) [inline content/Default/apps/content.json]: The title generally has to be under 14 characters or it won't fit under the icon on the desktop.

Comment (ptomato) [inline content/Default/apps/content.json]: @rmacqueen @MarielaBalbona Do we have any better title for this? This title won't fit under the desktop icon.

**Comment by rmacqueen** [comment]
ready

**Comment by ptomato** [comment]
**CR pass**

**Pull request merged**